R/filterFeatures.R

```diff
@@ -163,12 +161,17 @@ filterFeatures = function(task, method = "randomForestSRC_importance", fval = NU
       stopf("You supplied multiple filters. Please choose which should be used for the final subsetting of the features.")
     }
     if (is.null(select.method)) {
-      fval = fval[fval$method == fval$method, ]
+      fval = fval[fval$method == method[[1]], ]
     } else {
       assertSubset(select.method, choices = unique(fval$method))
       fval = fval[fval$method == select.method, ]
     }
   }
+  
+  if (select == "threshold") {
+    nselect = sum(fval[["value"]] >= threshold, na.rm = TRUE)
+  }
+
```

Comment:
This is a separate issue, isn't it?

Reply:
As above (it is the same issue) - it could be considered a separate issue, but it is part of the problem with ensemble filters not working correctly, which is why I included it. Happy to do a separate PR if you like.

Reply:
Please see #2685 (comment) regarding this.